spec/lib/engine/games/g_1849_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
```

Comment:
delete this

Reply:
rubocop complains and the other spec files also have this.

Reply:
i mean this whole file

Reply:
Ah, sure. I'll delete this and the game file and replace them with full games later.